include/site/linux

```diff
@@ -37,7 +37,7 @@ ac_cv_func_rindex=yes
 ac_cv_func_setlocale=yes
 ac_cv_func_setgrent_void=yes
 ac_cv_func_setpgrp_void=yes
-ac_cv_func_setresuid=no
+ac_cv_func_setresuid=yes
```

Comment:
IMHO it would be better to remove this line altogether than change its value.  Removing it would allow Autoconf to perform its checks as it normally would for any other `AC_CHECK_FUNCS()` call.

Forcing any value in this site script, be it `no` or `yes`, prevents a package `Makefile` from overriding it.  Removing it from this site script allows a package `Makefile` to set the value of this variable as it desires, if that ever becomes necessary to do explicitly.

AIUI, `setresuid()` should be treated identically as its sibling functions, e.g. `setresgid()`, which do not have any Autoconf cache values forced here.

Reply:
That raises the question, "Do we need this file at all?"

Reply:
I would also suggest to remove it here.

Reply:
Remote the entire file in this PR?

Reply:
Setting it to true should be fine.